src/main/java/hudson/plugins/audit_trail/AbstractLogFileAuditLogger.java

```diff
@@ -0,0 +1,124 @@
+package hudson.plugins.audit_trail;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.EnvVars;
+import hudson.Util;
+
+import javax.annotation.Nonnull;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.NoSuchFileException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Optional;
+import java.util.logging.FileHandler;
+import java.util.logging.Formatter;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.CONFIG;
+
+public abstract class AbstractLogFileAuditLogger extends AuditLogger {
+
+    private static final Logger LOGGER = Logger.getLogger(AbstractLogFileAuditLogger.class.getName());
+    static final String DEFAULT_LOG_SEPARATOR =" ";
+
+    @Nonnull
+    private String logSeparator;
+    private String log;
+    private int count = 1;
+    private int limit = 1;
```

Comment:
Limit is never used in `AbstractLogFileAuditLogger` and in `LogFileDailyRotationAuditLogger`, it's purely code pertaining to `LogFileAuditLogger`, it shouldn't appear elsewhere.